n2kMappings.js

```diff
@@ -12,6 +12,39 @@ function getMmsiContext(n2k) {
 }
 
 exports.mappings = {
+  //Battery Voltage
+  '127508': [{
+    source: 'Voltage',
+    node: 'electrical.batteries.0.capacity.stateOfCharge',
+    filter: function(n2k) {
+      return n2k.fields['Battery Instance'] === 0;
+    }
+  }, 
+{
+    source: 'Voltage',
+    node: 'electrical.batteries.1.capacity.stateOfCharge',
```

Comment:
It went pretty easy for me. I think the toughest part was figuring out the start and settings scripts. It would probably be nice to have some kind of configuration wizard/script.

Check out some scripts I use and my iOS app: https://github.com/sbender9/boat-scripts